src/components/Applications/Applications.tsx

```diff
@@ -114,10 +180,18 @@ class Applications extends Component<Props, State, {}> {
       {
         currentApplicationId: id,
         currentApplicationFilename: filename,
+        openModal: true,
       },
     );
+    console.log(this.state);
   }
 
+  placeholderFunc = () => { };
+
+  openModalFunc = () => this.setState({ openModal: true });
```

Comment:
just curious, why do we need two functions to open and close the modal? You could easily do the same thing with one function that will toggle the modal instead